### PR TITLE
fix: bun error on calling `jsr add` twice

### PR DESCRIPTION
**Pull request by nakasyou**
If you merged this, I think you may able to close https://github.com/jsr-io/jsr-npm/issues/35.

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export async function setupBunfigToml(dir: string) {
   const msg = `Setting up ${BUNFIG_FILE}`;
   try {
     let content = await fs.promises.readFile(bunfigPath, "utf-8");
-    if (!/^"@myorg1"\s+=/gm.test(content)) {
+    if (!/^"@myorg1"\s+=/gm.test(content) && !content.includes(JSR_BUNFIG)) {
```
Comment (marvinhagemeister) [inline src/commands.ts:53]: Let's only check if a `@jsr` registry mapping is in there.

```suggestion
   if (!/^"@jsr"\s+=/gm.test(content)) {
```

Comment (nakasyou) [reply src/commands.ts]: Thank you for reviewing! I see. I fixed it!

**Review by cena-ko — APPROVED** [review]

**Review by cena-ko — APPROVED** [review]

**Review by marvinhagemeister — COMMENTED** [review]
Sweet, let's add a test for this to ensure that we don't ever regress on this and then it's ready to be merged 👍

**Review by marvinhagemeister — APPROVED** [review]
Sweet, thanks for adding the test 👍

**Comment by marvinhagemeister** [comment]
Looks like the PR is not formatted correctly. Running `deno fmt` locally should resolve this.

**Comment by nakasyou** [comment]
Hi @marvinhagemeister, thank you for feedback!
I ran `deno fmt`.

I was confused about the formatter. There is a possibility that Other contributors will be confused just like me, so it might be a good idea to write description of formatter in a README.

**Pull request merged**